policy.go

```diff
@@ -120,10 +127,10 @@ func (n Nodes) Nodes() []uint32 {
 }
 
 // Weights returns slice ow nodes weights W.
-func (n Nodes) Weights() []uint64 {
-	w := make([]uint64, 0, len(n))
+func (n Nodes) Weights() []float64 {
```

Comment:
This function must return normalized weights since you updated HRW library `v1.0.7` -> `v1.0.8`.
It is used only once in `policy.go`
```go
hrw.SortSliceByWeightValue(nodes, nodes.Weights(), pivotHash)
```
This change might lead to the incorrect work of network map library with weighted nodes in some cases (tests are fine though). Do you plan to fix it in next pull requests or in this one?